src/DOM/HTML/Indexed/LengthLiteral.purs

```diff
@@ -0,0 +1,12 @@
+module DOM.HTML.Indexed.LengthLiteral where
+
+import Prelude (show, (<>))
+
+data LengthLiteral
+  = Pixels Int
```

Comment:
In my quick testing, browsers will convert this as necessary, so setting the property as an Int should work fine.